packages/resource-deployment/scripts/function-app-create.sh

```diff
@@ -67,6 +68,13 @@ installAzureFunctionsCoreTools() {
     esac
 }
 
+setReleaseVersionInTemplate() {
+    packageName=$1
+    templateFilePath="${0%/*}/../templates/function-$packageName-app-template.json"
+    parsedTemplateFilePath="${0%/*}/../templates/function-$packageName-app-template-parsed.json"
+    sed -e "s@%RELEASE_VERSION_NUMBER%@$releaseVersion@" "$templateFilePath" >"$parsedTemplateFilePath"
```

Comment:
we just need to pass value as parameter like below
`--parameters clientId="$webApiAdClientId" namePrefix="$functionAppNamePrefix" releaseVersion=... `